nifi-framework-bundle/nifi-framework/nifi-resources/pom.xml

```diff
@@ -116,6 +116,10 @@
         <nifi.status.repository.questdb.persist.component.days>3</nifi.status.repository.questdb.persist.component.days>
         <nifi.status.repository.questdb.persist.location>./status_repository</nifi.status.repository.questdb.persist.location>
 
+        <!-- NAR Persistence properties -->
+        <nifi.nar.persistence.provider.implementation>org.apache.nifi.nar.StandardNarPersistenceProvider</nifi.nar.persistence.provider.implementation>
+        <nifi.nar.persistence.provider.properties.directory>./nar_storage</nifi.nar.persistence.provider.properties.directory>
```

Comment:
I'm leaning towards `nar_repository` to be consistent with other repositories we have. I like something more specific than `nars` or `bundles` since it a special directory managed by the provider, as opposed to just a directory to drop NARs in.

Reply:
Yes, `nar_repository` sounds like a good option, following existing examples.